tests/dummy/app/templates/getting-started/assets.hbs

```diff
@@ -79,69 +79,66 @@ module.exports = function(defaults) {
-    <tr><th>print</th><td>@import "ember-bootstrap/print.less";</td><td>@import "ember-bootstrap/bootstrap/print.scss";</td><td>@import "ember-bootstrap/print.scss";</td></tr>
-    <tr><th>progress</th><td></td><td></td><td>@import "ember-bootstrap/progress.scss";</td></tr>
-    <tr><th>progress-bars</th><td>@import "ember-bootstrap/progress-bars.less";</td><td>@import "ember-bootstrap/bootstrap/progress-bars.scss";</td><td></td></tr>
-    <tr><th>reboot</th><td></td><td></td><td>@import "ember-bootstrap/reboot.scss";</td></tr>
-    <tr><th>responsive-embed</th><td>@import "ember-bootstrap/responsive-embed.less";</td><td>@import "ember-bootstrap/bootstrap/responsive-embed.scss";</td><td></td></tr>
-    <tr><th>responsive-utilities</th><td>@import "ember-bootstrap/responsive-utilities.less";</td><td>@import "ember-bootstrap/bootstrap/responsive-utilities.scss";</td><td></td></tr>
-    <tr><th>root</th><td></td><td></td><td>@import "ember-bootstrap/root.scss";</td></tr>
-    <tr><th>scaffolding</th><td>@import "ember-bootstrap/scaffolding.less";</td><td>@import "ember-bootstrap/bootstrap/scaffolding.scss";</td><td></td></tr>
-    <tr><th>tables</th><td>@import "ember-bootstrap/tables.less";</td><td>@import "ember-bootstrap/bootstrap/tables.scss";</td><td>@import "ember-bootstrap/tables.scss";</td></tr>
-    <tr><th>theme</th><td>@import "ember-bootstrap/theme.less";</td><td>@import "ember-bootstrap/bootstrap/theme.scss";</td><td></td></tr>
-    <tr><th>thumbnails</th><td>@import "ember-bootstrap/thumbnails.less";</td><td>@import "ember-bootstrap/bootstrap/thumbnails.scss";</td><td></td></tr>
-    <tr><th>tooltip</th><td>@import "ember-bootstrap/tooltip.less";</td><td>@import "ember-bootstrap/bootstrap/tooltip.scss";</td><td>@import "ember-bootstrap/tooltip.scss";</td></tr>
-    <tr><th>transitions </th><td></td><td></td><td>@import "ember-bootstrap/transitions.scss";</td></tr>
-    <tr><th>type</th><td>@import "ember-bootstrap/type.less";</td><td>@import "ember-bootstrap/bootstrap/type.scss";</td><td>@import "ember-bootstrap/type.scss";</td></tr>
-    <tr><th>utilities</th><td>@import "ember-bootstrap/utilities.less";</td><td>@import "ember-bootstrap/bootstrap/utilities.scss";</td><td>@import "ember-bootstrap/utilities.scss";</td></tr>
-    <tr><th>variables</th><td>@import "ember-bootstrap/variables.less";</td><td>@import "ember-bootstrap/bootstrap/variables.scss";</td><td>@import "ember-bootstrap/variables.scss";</td></tr>
-    <tr><th>wells</th><td>@import "ember-bootstrap/wells.less";</td><td>@import "ember-bootstrap/bootstrap/wells.scss";</td><td></td></tr>
-  </tbody>
-</table>
+<p>
+  It is possible to import only the assets your app needs but the definitions varies according to the chosen configuration.
+</p>
+
+<p><b>BS3 - Less</b></p>
+<pre><code class="language-less">// app.less
+
+// Required
+@import "ember-bootstrap/variables.less";
```

Comment:
I wonder if we should remove the file extension here and everywhere lese, `.less` as well as `.scss`?

I now this works for both. Always did it like this myself. BS4 also: https://github.com/twbs/bootstrap/blob/v4-dev/scss/bootstrap.scss

Reply:
Nice!